dbt/adapters/exceptions/compilation.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, missing: List):
         super().__init__(msg=self.get_message())
 
     def get_message(self) -> str:
-        msg = 'Snapshot target is not a snapshot table (missing "{}")'.format(
+        msg = 'Snapshot target is not a snapshot table or is missing configured columns (missing "{}")'.format(
```

Comment:
```suggestion
        msg = 'Snapshot target is missing meta-fields (missing "{}")'.format(
```
Maybe we also add a link to the docs? https://docs.getdbt.com/docs/build/snapshots#snapshot-meta-fields

Reply:
Added.